src/deployment/flavors.md

```diff
@@ -3,41 +3,171 @@ title: Creating flavors for Flutter
+(as shown in the screenshot), you’ll modify each scheme 
+to match its dev build configuration:
+
+![Step 5 Xcode image](/assets/images/docs/flavors/step5-ios-scheme.png){:width="100%"}
+
+## Using flavors in iOS
+
+Now that you’ve set up your development flavor, 
+you can, for example, add different product bundle identifiers per flavor. 
+A _bundle identifier_ uniquely identifies your application. 
+In this example, we set the **Debug-development** value to equal 
+`com.flavor-test.dev`. 
+
+&nbsp;1. Change the app bundle identifier to differentiate between schemes. 
+In **Product Bundle Identifier**, append `.dev` to each -development scheme value.
+
+![Step 1 using flavors image.](/assets/images/docs/flavors/step1-using-flavors.png){:width="100%"}  
+
+&nbsp;2. In the **Build Settings**, set the **Product Name** value to match each flavor. 
+
+![Step 2 using flavors image.](/assets/images/docs/flavors/step2-using-flavors.png){:width="100%"}  
+
+&nbsp;3. Add the display name to **info.plist**. Create a new key in **info.plist** called **Bundle Display Name** 
+with the value `$(PRODUCT_NAME)`. 
+
+![Step 3 using flavors image.](/assets/images/docs/flavors/step3-using-flavors.png){:width="100%"}    
+
+Now you have set up your development flavor by making a `development` scheme 
+in Xcode and setting the build configurations for that scheme. 
+
+For more information, skip to the [“Launching your app flavors”][] 
+section at the end of this document.
+
+### Plugin configurations
+
+If your app has a Flutter plugin, you will need to update the `ios/Podfile`. 
+
+* in `iOS/Podfile` change the default of **Debug**, **Profile**, and **Release** 
+to match the Xcode build configurations for the `development` scheme.
+
+![Podfile configuration image.](/assets/images/docs/flavors/podfile-config.png){:width="80%"}
+
+## Using flavors in Android
+
+Setting up flavors in Android can be done in your projects 
+**build.gradle** file.
+
+1. In your IDE, inside your Flutter project, 
+navigate to **android** > **app** > **build.gradle**.   
+
+![Step 1 using flavors in Android image.](/assets/images/docs/flavors/android-step1.png){:width="100%"}    
+
+Add a flavor object with the specified environments along with values for 
+**dimension**, **resValue**, and **applicationId**. 
+The name of the application for each build is located in **resValue** 
+(as shown in the screenshot below from VSCode).
+## Setting up launch configurations
+
+In VSCode, set up the launch configurations as follows:  
+Adding a **launch.json** file allows you to run the terminal command 
+`flutter run --flavor [environment name]`. 
+1. In the root directory of your project, add a folder called **.vscode**.    
+2. Inside the **.vscode** folder, create a file named **launch.json**.    
+3. In the **launch.json** file, add a configuration object for each flavor. 
+Each configuration has a **name**, **request**, **type**, **program**, 
+and **args** key.
```

Comment:
- Don't add text between the "...as follows:" line and the start of the list. It doesn't flow.
- You mention adding a hidden dot file, with ".vscode". Unless things have changed, this isn't how it works on Windows. @jmagman, how do you do this on Windows?

```suggestion
Next, you'll add a **launch.json** file so you can run the command 
`flutter run --flavor [environment name]` from a terminal.

In VSCode, set up the launch configurations as follows:  
1. In the root directory of your project, add a folder called **.vscode**.    
2. Inside the **.vscode** folder, create a file named **launch.json**.    
3. In the **launch.json** file, add a configuration object for each flavor. 
Each configuration has a **name**, **request**, **type**, **program**, 
and **args** key.
```